### PR TITLE
fix: plugin dir is empty

### PR DESCRIPTION
**Pull request by jim3ma**
Signed-off-by: Jim Ma <majinjing3@gmail.com>

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist:

- [ ] Code compiles correctly.
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by gaius-qi — APPROVED** [review]
LGTM

**Review by gaius-qi — DISMISSED** [review]
LGTM

File: cdn/cdn.go [hunk 1]
```diff
@@ -60,7 +62,10 @@ type Server struct {
 }
 
 // New creates a brand-new server instance.
-func New(cfg *config.Config) (*Server, error) {
+func New(cfg *config.Config, d dfpath.Dfpath) (*Server, error) {
```
Comment (244372610) [inline cdn/cdn.go:65]: source路径的更新可以做在RunE方法中吗？类似初始化日志路径。New()方法中只留下config参数

**Review by gaius-qi — APPROVED** [review]
LGTM

**Review by 244372610 — APPROVED** [review]

**Review by gaius-qi — APPROVED** [review]
LGTM

**Pull request merged**